prototypes/ItemRepeaterPrototypes/ItemsRepeaterExperiments/AttachedBehaviors/SelectionModelPage.xaml

```diff
@@ -0,0 +1,69 @@
+﻿<Page
+    x:Class="ItemsRepeaterExperiments.AttachedBehaviors.SelectionModelPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:ItemsRepeaterExperiments.AttachedBehaviors"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    xmlns:common="using:ItemsRepeaterExperiments.Common"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+        <muxc:ItemsRepeaterScrollHost Grid.Row="2">
+            <ScrollViewer HorizontalScrollBarVisibility="Auto" VerticalScrollBarVisibility="Auto">
+                <muxc:ItemsRepeater Grid.Row="2" 
+                                    x:Name="TeamPresenter"
+                    ElementPrepared="ItemsRepeater_ElementPrepared"
+                    ItemsSource="{x:Bind teams,Mode=OneWay}">
+
+                    <muxc:ItemsRepeater.Layout>
+                        <muxc:StackLayout Orientation="Horizontal"/>
+                    </muxc:ItemsRepeater.Layout>
+
+                    <muxc:ItemsRepeater.ItemTemplate>
+                        <DataTemplate x:DataType="common:Team">
+
+                                <!-- Inline UserControl -->
+                            <UserControl>
+                                <StackPanel x:Name="RootPanel" BorderBrush="Transparent" 
```

Comment:
I wonder if we can do a Binding to the attached property IsSelected from here and use a converter. Then we don't need the visual states at all ?

Reply:
Yes, that would also work.

Reply:
Totally agree, we can still support visual states, but don't require it.